### DISCUSSION TITLE
Vaultwarden won't update to 1.32.1

### DISCUSSION THREAD

**Discussion by niekniek89** [tteck/Proxmox#3832] [Help Needed]
Hello,

I have been running the following script for a while in Proxmox:

bash -c "$(wget -qO - https://github.com/tteck/Proxmox/raw/main/ct/alpine-vaultwarden.sh)"

This morning I see that there is an update (from 1.32. 0 to 1.32. 1).
When I restart the bash script and choose vaultwarden upgrade, it stays on the same version.
Even after a restart of the VM.
In terms of settings, I have never changed anything after using the script.
Who can help me on my way?

![image](https://github.com/user-attachments/assets/3becd15e-46ba-4b7b-8ad8-ea16382082d6)

**Comment by tteck** [comment]
Please learn how to use the search feature.
https://github.com/tteck/Proxmox/discussions?discussions_q=alpine+vaultwarden+
